packages/types/src/round.ts

```diff
@@ -4,9 +4,10 @@ import { removeTrailingZeros } from './shortify.js';
 export type RoundingMode = 'half-up' | 'up' | 'down';
 
 export interface RoundOptions {
-  value: number;
+  value: number | string;
```

Comment:
For @grod220 (because I know you want to know why) this is needed for a big number (caught by one of the tests)